demo/apiExamples.md

```diff
@@ -138,6 +145,44 @@ Sets a persistent error state (e.g. an error state returned from the server).
 
 </auro-accordion>
 
+#### noValidate
+
+Intended for use with the `required` attribute. If set, disables auto-validation on blur. By using these two attributes in combination the validation for required fields is still computed for forms but no validation messaging will be generated in the UI.
```

Comment:
```suggestion
Intended for use with the `required` attribute. When set, it disables `auto-validation` on blur. When using these two attributes in combination the validation for required fields is still computed for forms but no validation messaging will be generated in the UI.
```

Reply:
Language used here is cut and paste from the language in auro-input description of this attribute. I think we should keep these in sync.